apps/api/src/modules/notifications/schedule/scheduler.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
You can keep this in root directory or under a scripts folder in root directory as this is independent of the app. Would prefer the first case as we probably wont add more scripts and this is a temp solution.

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/293/commits/4a377834712185750ddc1dadd1d7408765aacbe9